src/units/unit.hpp

```diff
@@ -1737,11 +1737,11 @@ class unit : public std::enable_shared_from_this<unit>
 
 	/** Checks whether this unit currently possesses a given ability used like weapon
```

Comment:
Should the "used like weapon" be removed from this, as it appears to consider all abilities? IIUC, it's now "Checks whether this unit currently possesses a given ability, and that that ability is active.

Reply:
changed